bentoml/grpc/v1/service_test.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
```

Comment:
I know I said we should put this in as v1 but if we're releasing this as experimental we should probably actually call this an alpha release ;)

Reply:
v1alpha1?